jsonrpc/readiness.go

```diff
@@ -0,0 +1,56 @@
+package jsonrpc
+
+import (
+	"net/http"
+
+	"github.com/NethermindEth/juno/blockchain"
+	"github.com/NethermindEth/juno/sync"
+)
+
+const SyncBlockRange = 6
+
+type ReadinessHandlers struct {
+	bcReader   blockchain.Reader
+	syncReader sync.Reader
+}
+
+func NewReadinessHandlers(bcReader blockchain.Reader, syncReader sync.Reader) *ReadinessHandlers {
+	return &ReadinessHandlers{
+		bcReader:   bcReader,
+		syncReader: syncReader,
+	}
+}
+
+func (h *ReadinessHandlers) HandleReady(w http.ResponseWriter, _ *http.Request) {
+	w.WriteHeader(http.StatusOK)
+}
```

Comment:
Since this function always returns http.StatusOK, do you think it is a good idea to inline this function on line 215 in node/node.go?

Reply:
I asked about `/ready` endpoint in [tg](https://t.me/StarknetJuno/1224). Seems like we can safely remove it and just rely on / root path.